File: Rust/prog69.rs
use std::io::{self, Write};
fn main() {
    print!("Enter a number: ");
    io::stdout().flush().unwrap();
    let mut n1 = String::new();
    std::io::stdin()
        .read_line(&mut n1)
        .expect("Failed to read line");
    let n1: i32 = n1.trim().parse().expect("Please type a number!");
    println!("Double it and give it to the next person: {}", n1 * 2);
}

File: src/C#/prog69.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        while (true)
        {
            try
            {
                Console.Write("Enter a number: ");
                int n1 = int.Parse(Console.ReadLine());
                Console.WriteLine("Double it and give it to the next person: {0}", n1 * 2);
                break;
            }
            catch (FormatException)
            {
                Console.WriteLine("You must enter an integer. Please try again.");
            }
        }
    }
}
